packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -976,6 +976,14 @@ export class ContainerRuntime
 			compressionOptions.minimumBatchSizeInBytes !== Infinity &&
 			compressionOptions.compressionAlgorithm === "lz4";
 
+		// If both these settings are set this way, we will certainly not submit the GC op.
+		// In this case, we declare in Document Schema below that the gcOp feature is not supported.
+		// Once any client stops blocking GC op for a document, this schema bit will flip
+		// and clients that don't understand or unblock the GC op will face container closure on boot.
+		const gcOpBlocked =
```

Comment:
I would not do this here. Document schema should only tell whether GC ops is supported or not. This is passed to garbage collector which decides whether to send GC ops or not. That is a simpler model and is consistent with how other features are used.

Reply:
This should be done based on the `shouldRunSweep` flag. If that is enabled, we could send GC op and clients that do not understand this op should close.

Reply:
We can enable sweep + tombstone based on `shouldRunSweep`

Reply:
We paired on a better solution.